cron/monitoring/exporter.go

```diff
@@ -30,6 +30,8 @@ func NewStackDriverExporter() (*stackdriver.Exporter, error) {
 	exporter, err := stackdriver.NewExporter(stackdriver.Options{
 		ProjectID:    projectID,
 		MetricPrefix: "scorecard-cron",
+		// https://cloud.google.com/monitoring/quotas
```

Comment:
please add metrics name as well, there are many things on that page.

Reply:
Done.